notes.py

```diff
@@ -39,7 +39,8 @@ class NotesMod(loader.Module):
                "notes_header": "<b>Saved notes:</b>\n\n",
                "notes_item": "<b>•</b> <code>{}</code>",
                "delnote_args": "<b>What note should be deleted?</b>",
-               "delnote_done": "<b>Note deleted</b>"}
+               "delnote_done": "<b>Note deleted</b>", "delnotes_none": "<b>There are no notes to be cleared</b>",
```

Comment:
One per line please

Reply:
lmao i should've seen this coming

Reply:
not sure if i could edit in on github tho, might break lint